metals/src/main/scala/scala/meta/internal/metals/formatting/IndentOnPaste.scala

```diff
@@ -209,7 +209,7 @@ object IndentOnPaste extends RangeFormatter {
       ): String = {
         val identToStart = codeStartPosition(full).getOrElse(0)
 
-        if (identToStart != expectedIdent) {
+        if (identToStart < expectedIdent) {
```

Comment:
That works but I'm thinking maybe it would be more reliable if it won't do anything with indentation if paste changes only one non-empty line?

Reply:
Also, we have't covered all cases for `expectedIndent` calculation
In added test-case it's 2 while it should 4 because goes after `(`

Reply:
> That works but I'm thinking maybe it would be more reliable if it won't do anything with indentation if paste changes only one non-empty line?

Added additional logic for checking that

> Also, we have't covered all cases for expectedIndent calculation
In added test-case it's 2 while it should 4 because goes after (

Good catch, updated the regex and added tests.